streamwebs_frontend/streamwebs/templates/streamwebs/index.html

```diff
@@ -1,11 +1,8 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>StreamWebs</title>
-    </head>
-
-    <body>
-        <h1>StreamWebs</h1>
+{% extends 'streamwebs/base.html' %}
+{% load staticfiles %}
+{% load pipeline %}
```

Comment:
I think staticfiles and pipeline includes can be moved into base, they will be used in all of the child templates